### DISCUSSION TITLE
What languages do we know?

### DISCUSSION THREAD

**Discussion by BennyWestsyde** [UTD-4347-Databosses/Walmart_Database#2] [Polls]
We have encountered the first big question of our project. What backend do we want to use (#1)? 

To best handle this question, let's get an idea of what languages we know and are most comfortable with.

**Comment by BennyWestsyde** [comment]
Have you ever set up a website before? What backend did you use?

**Reply by GibbusMoon** [reply]
Firebase

**Reply by BeTheChange07** [reply]
I did but I used AWS light sale
